### PR TITLE
Refactor/update requirements

### PR DESCRIPTION
**Pull request by JanGut**
## Proposed changes
Cleanup of requirements.
(When merging, please squash this)

## Necessary checks
- [ ] ~~Update package version~~
- [ ] ~~Run linters~~
- [ ] ~~Run `catkin build`~~
- [ ] ~~Write documentation~~
- [x] Test on your machine
- [ ] ~~Test on the robot~~

### CODE REVIEW

**Comment by timonegk** [comment]
`breathe`, `construct` and `PyYAML` should be added.

**Comment by JanGut** [comment]
> but did you look through the packages if the `requirements3.txt` even contains current data?

No, I haven't checked all of them yet

**Comment by ftsell** [comment]
ping @timonegk please review because I changed stuff and therefore my review is kind of not useful

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+PyQt5
+PyYAML
+breathe
+construct
+matplotlib
+numpy
+opencv
```
Comment (timonegk) [inline requirements.txt]:
```suggestion
opencv-python
```

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+pip
+psutil
+pydot
+pyopencl
+rosdep
+rospkg
+setuptools
+silx
+tensorflow
+yaml
```
Comment (timonegk) [inline requirements.txt]: This package does not exist, PyYAML is the correct name.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+PyQt5
```
Comment (timonegk) [inline requirements.txt]: PyQt5 does not seem to be needed by any package

**Comment by JanGut** [comment]
@ftsell Did you found instances of `yaml` and `opencv` somewhere in the code? In this case, we should edit this.
And thank you for your additions!

**Comment by ftsell** [comment]
> @ftsell Did you found instances of `yaml` and `opencv` somewhere in the code? In this case, we should edit this.

I found it in package.xml files

**Pull request merged**